modules/odc-adding-a-GitHub-repository-containing-pipelines.adoc

```diff
@@ -51,3 +51,8 @@ image::Git-access-token-secret.png[]
 . Navigate back to the developer console and click *Add*.
 . Read the details of the steps that you have to perform and click *Close*.
 . View the details of the repository you just created.
+
+[NOTE]
+====
+When importing an application from Git using *Import from Git* in the *Developer* perspective, you can use the import strategy as `Pipeline-as-code` to create a Pipeline repository. For more information see, xref:../../applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc#odc-creating-applications-using-the-developer-perspective[Creating applications using the Developer perspective].
+====
```

Comment:
@shreyasiddhartha this is not accurate. There is no PAC strategy anymore. You might want to follow this story https://issues.redhat.com/browse/ODC-7234 instead. cc: @Lucifergene

Reply:
Hi @divyanshiGupta! Thanks for the update. I have made the changes again referring to the new story. Can you please review this?